### DISCUSSION TITLE
getServerSideProps() not being called.

### DISCUSSION THREAD

**Discussion by maximhorse314** [maximhorse314/optimetry-app#1] [Q&A] [answered]
Was trying to get familiar with nextJS 13. What I encountered is getServerSideProps function is not prerendering the page props. That's my first time trying it, so I don't know if I'm doing it wrong.

**Comment by coozebra** [comment, marked as answer]
It looks like you're trying to use getServerSideProps to perform server-side rendering and authentication checks before the page is displayed. From your code, it seems like you're on the right track.
